### DISCUSSION TITLE
One for work, the other at home

### DISCUSSION THREAD

**Discussion by joek** [george-norton/peacock#6] [Show and tell]
![image](https://github.com/george-norton/peacock/assets/202380/7e2bb7ec-e382-420b-8878-67ed15c7369e)
 
There is still the trackpad surface missing for the pink one but this will come soon.

**Comment by george-norton** [comment]
Great colours!

**Reply by MaxGust** [reply]
Is there a recommended AliExpress link for a trackpad surface?

**Reply by george-norton** [reply]
I have used this stuff on a couple of builds: https://www.aliexpress.com/item/32830509952.html
